conans/test/cmake_test.py

```diff
@@ -1,10 +1,18 @@
+import shutil
+import sys
+import tempfile
 import unittest
 from conans.model.settings import Settings
 from conans.client.conf import default_settings_yml
 from conans.client.cmake import CMake
 
 
 class CMakeTest(unittest.TestCase):
+    def setUp(self):
+        self.tempdir = tempfile.mkdtemp()
```

Comment:
Not even necessary to create/remove. Just use a random string, the SUT will not go to disk to check it, right? so no ``with`` context necessary?